lint failure

After the latest update of dependencies:

```
golangci-lint run --allow-parallel-runners
Error: exporter.go:87:12: SA1019: senders.NewProxySender is deprecated: Use 'senders.NewSender(url)' (staticcheck)
	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
	          ^
Error: metrics_exporter.go:32:12: SA1019: senders.NewProxySender is deprecated: Use 'senders.NewSender(url)' (staticcheck)
	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
	          ^
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### 🧰 Bug fixes 🧰
+
+- `tanzuobservabilityexporter`: remove calls to deprecated `NewProxySender` methods. (#11510)
+
 ## v0.54.0
 
 ## 🛑 Breaking changes 🛑

--- a/exporter/tanzuobservabilityexporter/exporter.go
+++ b/exporter/tanzuobservabilityexporter/exporter.go
@@ -70,7 +70,7 @@ func newTracesExporter(settings component.ExporterCreateSettings, c config.Expor
 	if !cfg.hasTracesEndpoint() {
 		return nil, fmt.Errorf("traces.endpoint required")
 	}
-	tracingHost, tracingPort, err := cfg.parseTracesEndpoint()
+	_, _, err := cfg.parseTracesEndpoint()
 	if err != nil {
 		return nil, fmt.Errorf("failed to parse traces.endpoint: %w", err)
 	}
@@ -84,13 +84,11 @@ func newTracesExporter(settings component.ExporterCreateSettings, c config.Expor
 
 	// we specify a MetricsPort so the SDK can report its internal metrics
 	// but don't currently export any metrics from the pipeline
-	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
-		Host:                 tracingHost,
-		MetricsPort:          metricsPort,
-		TracingPort:          tracingPort,
-		FlushIntervalSeconds: 1,
-		SDKMetricsTags:       map[string]string{"otel.traces.collector_version": settings.BuildInfo.Version},
-	})
+	s, err := senders.NewSender(cfg.Traces.Endpoint,
+		senders.MetricsPort(metricsPort),
+		senders.FlushIntervalSeconds(1),
+		senders.SDKMetricsTags(map[string]string{"otel.traces.collector_version": settings.BuildInfo.Version}),
+	)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create proxy sender: %w", err)
 	}

--- a/exporter/tanzuobservabilityexporter/metrics_exporter.go
+++ b/exporter/tanzuobservabilityexporter/metrics_exporter.go
@@ -28,14 +28,11 @@ type metricsExporter struct {
 	consumer *metricsConsumer
 }
 
-func createMetricsConsumer(hostName string, port int, settings component.TelemetrySettings, otelVersion string) (*metricsConsumer, error) {
-	s, err := senders.NewProxySender(&senders.ProxyConfiguration{
-		Host:                 hostName,
-		MetricsPort:          port,
-		DistributionPort:     port,
-		FlushIntervalSeconds: 1,
-		SDKMetricsTags:       map[string]string{"otel.metrics.collector_version": otelVersion},
-	})
+func createMetricsConsumer(endpoint string, settings component.TelemetrySettings, otelVersion string) (*metricsConsumer, error) {
+	s, err := senders.NewSender(endpoint,
+		senders.FlushIntervalSeconds(1),
+		senders.SDKMetricsTags(map[string]string{"otel.metrics.collector_version": otelVersion}),
+	)
 	if err != nil {
 		return nil, fmt.Errorf("failed to create proxy sender: %w", err)
 	}
@@ -53,7 +50,7 @@ func createMetricsConsumer(hostName string, port int, settings component.Telemet
 		true), nil
 }
 
-type metricsConsumerCreator func(hostName string, port int, settings component.TelemetrySettings, otelVersion string) (
+type metricsConsumerCreator func(endpoint string, settings component.TelemetrySettings, otelVersion string) (
 	*metricsConsumer, error)
 
 func newMetricsExporter(settings component.ExporterCreateSettings, c config.Exporter, creator metricsConsumerCreator) (*metricsExporter, error) {
@@ -64,11 +61,10 @@ func newMetricsExporter(settings component.ExporterCreateSettings, c config.Expo
 	if !cfg.hasMetricsEndpoint() {
 		return nil, fmt.Errorf("metrics.endpoint required")
 	}
-	hostName, port, err := cfg.parseMetricsEndpoint()
-	if err != nil {
+	if _, _, err := cfg.parseMetricsEndpoint(); err != nil {
 		return nil, fmt.Errorf("failed to parse metrics.endpoint: %w", err)
 	}
-	consumer, err := creator(hostName, port, settings.TelemetrySettings, settings.BuildInfo.Version)
+	consumer, err := creator(cfg.Metrics.Endpoint, settings.TelemetrySettings, settings.BuildInfo.Version)
 	if err != nil {
 		return nil, err
 	}

--- a/exporter/tanzuobservabilityexporter/metrics_exporter_test.go
+++ b/exporter/tanzuobservabilityexporter/metrics_exporter_test.go
@@ -63,7 +63,7 @@ func createMockMetricsExporter(
 	ourConfig := cfg.(*Config)
 	ourConfig.Metrics.Endpoint = "http://localhost:2878"
 	creator := func(
-		hostName string, port int, settings component.TelemetrySettings, otelVersion string) (*metricsConsumer, error) {
+		endpoint string, settings component.TelemetrySettings, otelVersion string) (*metricsConsumer, error) {
 		return newMetricsConsumer(
 			[]typedMetricConsumer{
 				newGaugeConsumer(sender, settings),

